server.js

```diff
@@ -868,6 +875,16 @@ module.exports.insertDevUser = function(callback) {
 
 if (config.startServer) {
     async.series([
+        function(callback) {
+            config.awsSetup(err => {
+                if (ERR(err, callback)) return;
+                callback(null);
+            });
+        },
+        function(callback) {
+            module.exports.appSetup();
```

Comment:
At this point, is it worth pulling it out to another file entirely? Scope the express.app stuff outside of server.js?